src/main/java/se/bjurr/gitchangelog/internal/issues/IssueParser.java

```diff
@@ -134,6 +137,7 @@ private ParsedIssue createParsedIssue(
     String title = "";
     final List<String> linkedIssues = new ArrayList<>();
     List<String> labels = new ArrayList<>();
+    final Map<String, Object> additionalFields = new HashMap<>();
```

Comment:
I prefer `TreeMap` to get deterministic results, not random iteration order.

Reply:
Good call out, done!